basicsr/archs/hifacegan_arch.py

```diff
@@ -0,0 +1,265 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+# 20210519 [Lotayou]: Currently Pytorch official SyncBatchNorm only works under DDP training.
+# from basicsr.models.archs.sync_batchnorm import SynchronizedBatchNorm2d
```

Comment:
as this file is not used, just remove `from basicsr.models.archs.sync_batchnorm import SynchronizedBatchNorm2d`